### ISSUE TITLE
Allow identical lines in import

### ISSUE THREAD

**Issue by jzohrab** [LuteOrg/lute-v3#522]
Currently the term import fails if the same term is listed twice, e.g. in tests/features/term_import:

```
    Scenario: Duplicate term throws
        Given import file:
            language,term
            Spanish,gato
            Spanish,gato
        Then import should fail with message:
            Duplicate terms in import: Spanish: gato
```

Really, this should only be a problem if there is inconsistent data.  e.g, this is bad:

```
    Scenario: Duplicate term throws
        Given import file:
            language,term,translation
            Spanish,gato,cat
            Spanish,gato,kitty
        Then import should fail with message:
            Duplicate terms in import: Spanish: gato
```

but this should work:

```
    Scenario: identical lines ok
        Given import file:
            language,term
            Spanish,gato,cat
            Spanish,gato,cat
        Then import should succeed with 1 term imported
```

**Comment by jzohrab** [comment]
fixed in `develop` with test case.

**Comment by jzohrab** [comment]
Released in 3.6.1.
